src/action-configuration.ts

```diff
@@ -34,6 +35,7 @@ export interface ICreateOrUpdateActionParams {
     cpu: number;
     memory: number;
     environment?: Record<string, string>;
+    autoDeploymentsEnabled: boolean
```

Comment:
what is the reason for having a copy of that configuration parameter at the root level, although the only one defined on `ICodeConfiguration` interface is being used by the rest of the code?